.github/workflows/documentation.yml

```diff
@@ -13,14 +13,14 @@ jobs:
 
     steps:
       - name: Checkout branch
-        uses: actions/checkout@v2
+        uses: actions/checkout@v4
         with:
           token: ${{ secrets.CI_PAT }}
           fetch-depth: 0
 
       - name: Clear docs
         run: |
-          rm -r /tmp/triton-docs
+          rm -rf /tmp/triton-docs
         continue-on-error: true
```

Comment:
I don't think this line is needed anymore, but it doesn't hurt anything either. I assume it was a workaround to unblock the build from [this error](https://github.com/openai/triton/actions/runs/8405337316/job/23017918791#step:3:6) since it didn't matter to the rest of the flow. -f ensures the rm command always returns success.

> Run rm -r /tmp/triton-docs
rm: cannot remove '/tmp/triton-docs': No such file or directory
Error: Process completed with exit code 1.